lib/logos.js

```diff
@@ -88,25 +88,38 @@ function getSimpleIconStyle({ icon, style }) {
   return 'default'
 }
 
-function getSimpleIcon({ name, color, style }) {
+function getSimpleIcon({ name, color, style, size = 'contain' }) {
```

Comment:
It is odd that the default value here is "contain", but then we never reference "contain" anywhere (because it basically means "don't change anything").

First, I wonder if something like "square" might be a better name for the default (open to suggestions on this), or we could just make the default `null`?

Second, can we add a comment in the function explaining what the 2 possible values are and what they mean.